Application/EdFi.Ods.WebApi/StartupFactory.cs

```diff
@@ -0,0 +1,51 @@
+﻿// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+using System.Linq;
+using EdFi.Ods.Api.Startup;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using System;
+using System.IO;
+
+namespace EdFi.Ods.WebApi
+{
+    public static class StartupFactory
+    {
+       
+        private const string MissingStartupClassMessage =
+            "Could not find the startup class. No loaded class member could be found matching the configured startup class name '{0}'.";
```

Comment:
Change this message to:
`Could not find a startup class named '{0}' that derives from OdsStartupBase.`